backend/kernelCI_app/views/bootsView.py

```diff
@@ -0,0 +1,127 @@
+from typing_extensions import Union
+from django.http import JsonResponse
+from django.views import View
+from kernelCI_app.models import Checkouts
+import json
+
+
+class Boots(View):
+    # TODO misc_environment is not stable and should be used as a POC only
+    # use the standardized field when that gets available
+    def extract_platform(self, misc_environment: Union[str, dict]):
+        parsedEnvMisc = None
+        if isinstance(misc_environment, dict):
+            parsedEnvMisc = misc_environment
+        else:
+            parsedEnvMisc = json.loads(misc_environment)
+        platform = parsedEnvMisc.get("platform")
+        if platform:
+            return platform
+        print("unknown platform in misc_environment", misc_environment)
+        return "unknown"
+
+    # TODO misc is not stable and should be used as a POC only
+    def extract_error_message(self, misc: Union[str, dict]):
+        parsedEnv = None
+        if isinstance(misc, dict):
+            parsedEnv = misc
+        else:
+            parsedEnv = json.loads(misc)
+        error_message = parsedEnv.get("error_msg")
+        if error_message:
+            return error_message
+        print("unknown error_msg in misc", misc)
+        return "unknown"
+
+    def get(self, request, commit_hash: str | None):
+        print("commit_hash", commit_hash)
```

Comment:
left over?

Reply:
solved in https://github.com/kernelci/dashboard/pull/61/commits/db5a19c28bcf9c8186de01740beea10901e84386